src/tools/ceph_monstore_tool.cc

```diff
@@ -532,6 +532,13 @@ static int update_auth(MonitorDBStore& st, const string& keyring_path)
     KeyServerData::Incremental auth_inc;
     auth_inc.name = k.first;
     auth_inc.auth = k.second;
+    if (auth_inc.auth.caps.empty()) {
```

Comment:
to be specific, i would like ceph-monstore-tool to be a flexible low level tool with minimum assumptions. and the functionality to import keyrings when building the mon-store follows the philosophy.